### PR TITLE
test: improve admin coverage

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

**Comment by joschkabraun** [comment]
Admin coverage is very low: https://app.codecov.io/gh/jina-ai/now/pull/1210/tree/now/admin

**Comment by joschkabraun** [comment]
Are you sure the exclusion of `admin` works https://app.codecov.io/gh/jina-ai/now/pull/1210/tree/now?

**Comment by IyadhKhalfallah** [comment]
> Are you sure the exclusion of `admin` works https://app.codecov.io/gh/jina-ai/now/pull/1210/tree/now?

It should be ignored when adding the folder to the list of ignored coverages. I'm not sure why it is not doing that, I'll look into it

File: now/admin/benchmark_flow.py
Comment (joschkabraun) [inline now/admin/benchmark_flow.py:1]: I am rewrite this file in another PR, so we can drop these changes.

**Review by joschkabraun — COMMENTED** [review]
Nice bump in test accuracy!
Small suggestion.

**Review by joschkabraun — APPROVED** [review]
LGTM!

**Review by Tanguyabel — APPROVED** [review]

**Pull request merged**